docs/docsite/rst/playbook_guide/playbooks_loops.rst

```diff
@@ -268,7 +268,7 @@ To see the results of individual retries, run the play with ``-vv``.
 
 When you run a task with ``until`` and register the result as a variable, the registered variable will include a key called "attempts", which records the number of the retries for the task.
 
-.. note:: You must set the ``until`` parameter if you want a task to retry. If ``until`` is not defined, the value for the ``retries`` parameter is forced to 1.
```

Comment:
I wasn't sure how docs versioning works now docs is in its own repo, should the `note` stay and the new information be guarded with `version_added: 2.15`?

Reply:
In general, the no-backport label should ensure this doesn't go anywhere but devel until 2.16 releases.